ansible_ai_connect/users/migrations/0014_auto_20240829_1738.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.15 on 2024-08-29 17:38
+
+from django.db import migrations
+
+
+def email_names_from_db(apps, schema_editor):
+    User = apps.get_model("users", "User")
+    for user in User.objects.all():
+        if user.first_name:
+            continue
+        ams = user.get("ams")
+        if not ams:
+            continue
+        user.first_name = ams.get("first_name")
```

Comment:
You're using `given_name` on your other [PR](https://github.com/ansible/ansible-ai-connect-service/pull/1281/files#diff-d8b9627bf3e9cc38433bebfcb64ea46bc5dee93c7eaa4e4fc62fe6c3fa1d4341R71).